metrics/web/lib/dashboard/presentation/pages/dashboard_page.dart

```diff
@@ -44,27 +45,30 @@ class _DashboardPageState extends State<DashboardPage> {
 
   @override
   Widget build(BuildContext context) {
-    return MetricsScaffold(
-      body: Column(
-        children: <Widget>[
-          Padding(
-            padding: const EdgeInsets.only(bottom: 60.0),
-            child: Row(
-              children: <Widget>[
-                Flexible(
-                  child: ProjectMetricsSearchInput(),
-                ),
-                Padding(
-                  padding: const EdgeInsets.only(left: 32.0),
-                  child: ProjectGroupsDropdownMenu(),
-                ),
-              ],
+    return PageParametersProxy(
+      pageNotifier: _projectMetricsNotifier,
```

Comment:
:umbrella_with_rain_drops:
What do you think about adding this widget into the `MetricsScaffold` to avoid duplication on each page?